File: library/src/main/java/com/nshmura/strictmodenotifier/StrictModeLog.java
package com.nshmura.strictmodenotifier;

public class StrictModeLog {
  public final String tag;
  public final String message;
  public final long time;

  public StrictModeLog(String tag, String message, long time) {
    this.tag = tag;
    this.message = message;
    this.time = time;
  }

  public boolean isAt() {
    return message.matches("^\\s+at.*");
  }
}

File: notifier/src/main/java/io/github/juanimoli/strictmode/notifier/StrictModeLog.kt
package io.github.juanimoli.strictmode.notifier

class StrictModeLog(val tag: String, val message: String, val time: Long) {
    val isAt: Boolean
        get() = message.matches("^\\s+at.*".toRegex())
    val isCausedBy: Boolean
        get() = message.matches("^\\s+Caused by.*".toRegex())
}
